reverse/reverse.py

```diff
@@ -0,0 +1,10 @@
+def reverse(word):
+    if word:
```

Comment:
I actually don't think you have to check if the string has a length, because it will just return `new_word` and not go into the `for` loop.